x-pack/plugins/enterprise_search/server/lib/enterprise_search_request_handler.ts

```diff
@@ -22,10 +25,9 @@ import {
   READ_ONLY_MODE_HEADER,
 } from '../../common/constants';
 
-import { ConfigType } from '../index';
+import { entSearchHttpAgent } from './enterprise_search_http_agent';
 
 interface ConstructorDependencies {
-  config: ConfigType;
```

Comment:
This should not have been removed it looks like, it's still a passed param/dependency.

LMK if you want any help fixing the current type errors - happy to pair or push something up